deploy/script/katello-debug.sh

```diff
@@ -61,6 +61,13 @@ fi
 add_files /etc/gofer
 add_files /var/log/gofer
 
+#foreman-tasks export
+if hash foreman-rake  2>/dev/null; then
+  echo "Exporting tasks, this may take a few minutes."
+  tasks_filename=`foreman-rake foreman_tasks:export_tasks | tail -n 1 | awk '{print $2}'`
```

Comment:
Its awaiting a new build of foreman-tasks:

https://github.com/theforeman/foreman-tasks/commit/3755354bea374cd230bd5b4a05e8c662ed4fb5de